soc/arm/rpi_pico/rp2/Kconfig.defconfig.series

```diff
@@ -0,0 +1,17 @@
+# Raspberry Pi RP2XXX MCU line
+
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
Nordic?

Reply:
> Nordic?

Just a note that if it's copy/pasted from something with Nordic copyright, that should not be removed, although additional copyrights may be added if there are changes, of course.

Reply:
This file was originally created by @ioannisg , I left the copyright as it was.